_posts/2020-12-21-sidekiq-incident-learnings.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title:  "Learning from incidents: getting Sidekiq ready to serve a billion jobs"
+author: nakulpathak3
+tags:
+- incident response
+- sidekiq
+- monitoring
+team: Internal Tools
+---
+
+Scribd currently serves hundreds of Sidekiq jobs per second and has served 25 billion jobs since its adoption. Getting to this scale wasn’t easy. In this post, I’ll walk you through one of our first ever Sidekiq incidents and how we improved our Sidekiq implementation as a result of this incident.
```

Comment:
@cktaylor I'm thinking maybe we should specify `since its adoption 1.5 years ago` for clarity.

Reply:
Added 2 years ago